### PR TITLE
refactor: consolidate TopNav across pages, minor UI tweaks

### PR DESCRIPTION
**Pull request by 0xBigBoss**
- Consolidate the `HomeTopNav` and `AccountTopNav` components into a single reusable `TopNav` component. The new `TopNav` takes props to customize what buttons/elements are shown.
- Update the pages and screens to use the new consolidated `TopNav` component instead of the page-specific top navs. Pass in the desired props to configure the header, subheader, and buttons.
- Make some minor styling tweaks and adjustments to margins/padding in a few places, especially on larger screen sizes.
- Remove an unused /account/profile page.
- Add a new "jet" color to the theme.

So in summary, it's primarily a refactor to consolidate redundant top navigation code into a single reusable component, along with a handful of small UI style tweaks and cleanup. The new `TopNav` component allows the top nav to be more easily customized per page while sharing the core layout and functionality.

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
Nice I'm glad to see this change. Will review right now

File: packages/app/components/TopNav.tsx [hunk 1]
```diff
@@ -11,34 +12,53 @@ import {
...
+  showLogo?: boolean
+  showQrButton?: boolean
+  showSettingsButton?: boolean
+}
+
+export function TopNav({
+  header,
+  subheader,
+  showLogo = false,
+  showQrButton = false,
```
Comment (youngkidwarrior) [inline packages/app/components/TopNav.tsx:32]: Not the biggest fan of this implementation, since these will never bot be true

Instead we can have a component prop or a string enum like
```
  button: "settings" | "qr"
```

**Review by youngkidwarrior — APPROVED** [review]
Just a suggestion for a cleaner button implementation

File: packages/app/components/TopNav.tsx (hunk 1 above)
Comment (0xBigBoss) [reply packages/app/components/TopNav.tsx:32]: great suggestion
